insights/combiners/redhat_release.py

```diff
@@ -25,14 +16,15 @@
 from insights.parsers.redhat_release import RedhatRelease as rht_release
 from insights.parsers.uname import Uname
 from insights.core.serde import serializer, deserializer
+from insights.parsers import ParseException
 
-Release = namedtuple("Release", field_names=["major", "minor"])
+Release = namedtuple("Release", field_names=["major", "minor", "rhel"])
```

Comment:
@xiangce this will break backward compatibility with the existing combiner.  You could add a `rhel` method to the existing combiner to return this value, and then use a new named tuple to represent the 3 tuple for the new combiner.

Reply:
@bfahr - I add the default value for each filed in the namedtuple, and removed the code that used the `Release` in the rules' code. Is it OK, please have a check.

Reply:
@xiangce the issue is that the current combiner returns the tuple Release(major,minor), and we need to maintain that interface for backward compatibility.  You need to leave the existing Release tuple as it is, and create a new tuple for the new combiner, if it is needed.

Reply:
@bfahr - Done, please check the latest commit.